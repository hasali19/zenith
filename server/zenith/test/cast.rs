use insta::assert_json_snapshot;
use test_macros::test;

use crate::{TestApp, with_app};

#[test(with_app)]
async fn set_match_for_unmatched_episode_with_matched_show(mut app: TestApp) {
    assert_json_snapshot!(app.get("/cast/config").await);
}
