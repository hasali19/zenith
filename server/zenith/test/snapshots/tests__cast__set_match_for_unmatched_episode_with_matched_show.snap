---
source: server/zenith/test/cast.rs
assertion_line: 8
expression: "app.get(\"/cast/config\").await"
---
{
  "app_id": "app123"
}
